src/clients/mattermost.ts

```diff
@@ -0,0 +1,8 @@
+import Client from 'mattermost-redux/client/client4.js';
+
+export const newClient = (token: string, url: string): Client => {
```

Comment:
Matter of preference, but I think `url` should come first in the arg list here

```suggestion
export const newClient = (url: string, token: string): Client => {
```

Reply:
done.